From 5a2779f9bc829bdd804e68d54c14ec61d07d0295 Mon Sep 17 00:00:00 2001
From: "H.J. Lu" <hjl.tools@gmail.com>
Date: Fri, 27 Oct 2017 14:09:01 -0700
Subject: [PATCH 112/146] i386: Regenerate libm-test-ulps for for gcc 7

Regenerate libm-test-ulps for gcc 7 with "-m32 -O2 -march=i586".

	* sysdeps/i386/fpu/libm-test-ulps: Regenerated for GCC 7 with
	"-O2 -march=i586".

(cherry picked from commit 63d3b468c1b63d04c067eab8d86f0be6bd15bc87)
---
 ChangeLog                       |  5 +++++
 sysdeps/i386/fpu/libm-test-ulps | 12 ++++++------
 2 files changed, 11 insertions(+), 6 deletions(-)

diff --git a/ChangeLog b/ChangeLog
index b90e7eb827..1bf692af23 100644
--- a/ChangeLog
+++ b/ChangeLog
@@ -1,3 +1,8 @@
+2017-10-27  H.J. Lu  <hongjiu.lu@intel.com>
+
+	* sysdeps/i386/fpu/libm-test-ulps: Regenerated for GCC 7 with
+	"-O2 -march=i586".
+
 2017-12-13  Adhemerval Zanella  <adhemerval.zanella@linaro.org>
 
 	* sysdeps/ia64/fpu/libm-test-ulps: Update.
diff --git a/sysdeps/i386/fpu/libm-test-ulps b/sysdeps/i386/fpu/libm-test-ulps
index ed685de35d..a269c7c343 100644
--- a/sysdeps/i386/fpu/libm-test-ulps
+++ b/sysdeps/i386/fpu/libm-test-ulps
@@ -2693,30 +2693,30 @@ ldouble: 2
 
 Function: "y1_downward":
 double: 2
-float: 2
+float: 3
 float128: 4
 idouble: 2
-ifloat: 2
+ifloat: 3
 ifloat128: 4
 ildouble: 7
 ldouble: 7
 
 Function: "y1_towardzero":
 double: 2
-float: 2
+float: 3
 float128: 2
 idouble: 2
-ifloat: 2
+ifloat: 3
 ifloat128: 2
 ildouble: 5
 ldouble: 5
 
 Function: "y1_upward":
 double: 1
-float: 2
+float: 3
 float128: 5
 idouble: 1
-ifloat: 2
+ifloat: 3
 ifloat128: 5
 ildouble: 7
 ldouble: 7
-- 
2.15.1

